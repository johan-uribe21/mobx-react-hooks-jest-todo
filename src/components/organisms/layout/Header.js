import React from "react";
import styled, { css } from "styled-components";

const HeaderBar = styled.div`
  background: transparent;
  border-radius: 1px;
  border: 1px solid palevioletred;
  color: palevioletred;
  margin: 0px;
  padding: 5px;
  width: 100%;
  length: 100px;
  display: flex;
  flex-directly: row;
  justify-content: center;
  align-items: center;
`;

export default function Header() {
  return (
    <div style={{ width: "100%" }}>
      <HeaderBar>
        <div>Trello Clone Header Bar</div>
      </HeaderBar>
    </div>
  );
}
