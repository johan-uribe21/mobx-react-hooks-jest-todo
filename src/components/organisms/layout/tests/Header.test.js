import React from "react";
import { render } from "@testing-library/react";
import renderer from "react-test-renderer";
import Header from "../Header";

test("renders 'Trello Clone Header Bar'", () => {
  const { getByText } = render(<Header />);
  const divElement = getByText("Trello Clone Header Bar");
  expect(divElement).toBeInTheDocument();
});

test("Header matches snapshot", () => {
  const component = renderer.create(<Header />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
