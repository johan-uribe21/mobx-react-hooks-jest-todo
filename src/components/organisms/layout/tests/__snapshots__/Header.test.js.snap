// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header matches snapshot 1`] = `
<div
  style={
    Object {
      "width": "100%",
    }
  }
>
  <div
    className="sc-AxjAm kzHKfW"
  >
    <div>
      Trello Clone Header Bar
    </div>
  </div>
</div>
`;
